helm/dns-operator-azure/values.schema.json

```diff
@@ -30,6 +30,19 @@
         "baseDomain": {
             "type": "string"
         },
+        "global": {
+            "type": "object",
+            "properties": {
+                "podSecurityStandards": {
+                    "type": "object",
+                    "properties": {
+                        "enforced": {
+                            "type": "boolean"
```

Comment:
Would it better to add a default here? What do you think?

Reply:
We haven't added it to other apps but it makes sense. Added here.